src/main/java/module/youth/YouthPlayerOverviewTableModel.java

```diff
@@ -76,7 +76,7 @@ public IHOTableEntry getTableEntry(YouthPlayer player) {
                 new YouthPlayerColumn("ls.youth.player.average") {
                     @Override
                     public IHOTableEntry getTableEntry(YouthPlayer player) {
-                        return new ColorLabelEntry(player.getAverageSkillLevel(), ColorLabelEntry.FG_STANDARD, ColorLabelEntry.BG_STANDARD, SwingConstants.RIGHT);
+                        return new ColorLabelEntry(player.getAllrounderSkillLevelAsString(), ColorLabelEntry.FG_STANDARD, ColorLabelEntry.BG_STANDARD, SwingConstants.RIGHT);
```

Comment:
Out of curiousity, what does "all-rounder" mean in this context?

Reply:
Just happened: 

Scout Comment
I think I have found a skilled youngster for you. His name is Thorsten Tilschner and he is 15 years old.

Given the chance to improve his Playmaking skills, this guy might well reach passable in that department. Right now I would say his Winger capabilities are around the inadequate level. With his inadequate **overall skills, t**his might be the player for us. 


@tychobrailleur Might be a good idea to rename it to OverallSkills

Reply:
> @tychobrailleur Might be a good idea to rename it to OverallSkills

Ah yes, that actually makes more sense to me.  Thanks!